src/Configuration/Definition.php

```diff
@@ -22,13 +22,25 @@ public function getConfigTreeBuilder(): TreeBuilder
                     ->scalarPrototype()->end()
                 ->end()
                 ->arrayNode('analyzer')
+                    ->setDeprecated('qossmic/deptrac', '0.14.1', 'The "%node%" option is deprecated. Use "analyser" instead.')
```

Comment:
```suggestion
                    ->setDeprecated('qossmic/deptrac', '0.15', 'The "%node%" option is deprecated. Use "analyser" instead.')
```
We are already at `0.14.1`, so next release will likely be `0.15`

Reply:
Done